src/iframily.js

```diff
@@ -7,26 +7,66 @@ const constants = require('./constants');
 let parentIframilies = {};
 let childIframilies = {};
 
+function validateIdAvailable(id, iframilies, type) {
+    if (iframilies[id] && !iframilies[id].disposed) {
+        // eslint-disable-next-line max-len
+        console.error(`[Iframily] - A ${type} iframily with id "${id}" was already inited, please use another id or dispose the existing one first.`);
+        return false;
+    }
+
+    return true;
+}
+
+function validateTargetOrigin(targetOrigin, id) {
+    if (!targetOrigin) {
+        console.error(`[Iframily] - Missing "targetOrigin" argument, not initing "${id}" iframily id.`);
+        return false;
+    }
+
+    if (typeof targetOrigin !== 'string') {
+        // eslint-disable-next-line max-len
+        console.error(`[Iframily] - "targetOrigin" is of type ${typeof targetOrigin} but must be of type string, not initing "${id}" iframily id.`);
+        return false;
+    }
+
+    if (targetOrigin === '*') {
+        // eslint-disable-next-line max-len
+        console.error(`[Iframily] - "*" (wildcard) is not allowed for "targetOrigin" argument. If you are sure about what you are doing, use "${constants.DANGEROUSLY_SET_WILDCARD}".`);
```

Comment:
Add link to docs

Reply:
Do you imagine the link to be to the github README url of that section? like so?
https://github.com/EkoLabs/iframily#----iframily-singleton----

Reply:
No just something like 'e.g.  "https://eko.com' to remove friction so that we don't have a link pointing to a link with docs

Reply:
you mean "https://github.com/EkoLabs/iframily" ?